app/js/lib/foxbox/services/base.js

```diff
@@ -151,6 +153,80 @@ export default class BaseService extends EventDispatcher {
   }
 
   /**
+   * Returns list of service tags.
+   *
+   * @return {Array<string>}
+   */
+  getTags() {
```

Comment:
Made it a method instead of property, to make tag management less confusing (when you have an array property it may seem that just pushing/popping an element to this list will add/remove tag).